examples/layers/JSONLayers/Cada.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
Why do you add a new example layer in a `fix` PR?

Reply:
why not

Reply:
Seriously :-/ ?